crypto/fipsmodule/evp/evp.c

```diff
@@ -543,12 +543,11 @@ int EVP_PKEY_CTX_get_signature_md(EVP_PKEY_CTX *ctx, const EVP_MD **out_md) {
 }
 
 void *EVP_PKEY_get0(const EVP_PKEY *pkey) {
-  // Node references, but never calls this function, so for now we return NULL.
-  // If other projects require complete support, call |EVP_PKEY_get0_RSA|, etc.,
-  // rather than reading |pkey->pkey.ptr| directly. This avoids problems if our
-  // internal representation does not match the type the caller expects from
-  // OpenSSL.
-  return NULL;
+  GUARD_PTR(pkey);
```

Comment:
not a request for a change, just an observation -- coverity shows this PR as having 100% patch coverage, but i don't see a test case for `EVP_PKEY_get0(NULL)`. i guess that implies that macro-generated code isn't included in coverage data?

Reply:
I added a test case for `EVP_PKEY_get0(NULL)` in the latest commit.